ui/pages/teams.vue

```diff
@@ -1,21 +1,52 @@
 <template>
   <v-container>
-    <v-layout column>
-      <v-flex>
-        <v-layout column align-center>
-          <page-title title="チーム一覧" />
-        </v-layout>
-      </v-flex>
-    </v-layout>
+    <v-row justify="center">
+      <page-title title="チーム一覧" />
+    </v-row>
+​
+    <v-divider class="mt-3 mb-4" />
+​
+    <v-row align="start" class="mx-5" no-gutters>
+      <v-col
+        v-for="team of teams"
+        :key="team.id"
+        :cols="isFixedWidth ? 'auto' : 4"
+      >
+        <v-card tile class="mb-5 mr-6">
+          <v-card-title class="subtitle-1">
+            {{ team.name }}
```

Comment:
```suggestion
            {{ team.displayName }}
```

チーム番号も一緒に表示したいのでこれでお願いします。